src/containers/Drawer/ResponsiveDrawer.module.css

```diff
@@ -0,0 +1,5 @@
+.brand_name {
+  font-weight: 100;
+  padding: 8px;
+  border-bottom: 3px solid #2089dc;
```

Comment:
We can replace this with a <Divider> from mui/core

Reply:
Ok, I will add those styles from css module to makeStyles of material-ui/core/styles